scipy/optimize/tests/test_zeros.py

```diff
@@ -58,5 +59,17 @@ def test_deriv_zero_warning(self):
         assert_warns(RuntimeWarning, cc.newton, func, 0.0, dfunc)
 
 
+class TestRootResults:
```

Comment:
Don't you need to inherit from `unittest.TestCase`? If not, at least, it'd be clearer and less dependent on the test runner.